x-pack/plugins/index_management/public/application/components/template_form/template_form.tsx

```diff
@@ -292,7 +292,7 @@ export const TemplateForm = ({
   return (
     <>
       {/* Form header */}
-      {title}
+      <EuiPageHeader pageTitle={<span data-test-subj="pageTitle">{title}</span>} bottomBorder />
```

Comment:
Should we ask the EUI team to add a generic `data-test-subj` for us so we don't have to provide everywhere a `<span>` for the title? We should only have one page header per page right?

Reply:
Good idea. They already have a generic `data-test-subj`, but we're excluding the work of migrating our code and tests over to it from the scope of this work due to time pressure.